packages/zowe-explorer/src/utils/TempFolder.ts

```diff
@@ -104,12 +105,16 @@ export function cleanDir(directory) {
  * @export
  */
 export async function cleanTempDir() {
+    // Get temp folder location from settings
```

Comment:
@JillieBeanSim The comment in this line doesn’t seem to describe what is actually happening in the block below it. Should it maybe say `// Get temp folder cleanup preference from settings`?